services/movie.py

```diff
@@ -0,0 +1,36 @@
+from django.db.models import QuerySet
+from db.models import Movie
+
+
+def get_movies(
+    genres_ids: list[int] | None = None, actors_ids: list[int] | None = None
+) -> QuerySet:
+    result = Movie.objects.all()
+    if not genres_ids and not actors_ids:
+        return result
+    if genres_ids:
+        result = result.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        result = result.filter(actors__id__in=actors_ids)
+    return result
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)

```

Comment:
Using `Movie.objects.get(id=movie_id)` will raise a `DoesNotExist` exception if the movie with the given ID is not found. Consider using `get_object_or_404` if this function is used in a view, or handle the exception appropriately.